### PR TITLE
Add maintainers and version info to AWS Docker image

### PR DESCRIPTION
**Pull request by GiaJordan**
Resolves FDS-674

### CODE REVIEW

**Comment by linglp** [comment]
@GiaJordan here it looks like that there are a lot of changes. What are the actual changes?

File: schematic_api/Dockerfile [hunk 1]
```diff
@@ -1,120 +1,126 @@
...
+    APP_DIR=/app/app \
+    ROOT=/ \
+    UWSGI_INI=/app/uwsgi.ini \
+    NGINX_WORKER_PROCESSES=1 \
+    VERSION=$TAG
+
+
+
+LABEL maintainer="Lingling Peng <lingling.peng@sagebase.org> Andrew Lamb <andrew.lamb@sagebase.org> Gianna Jordan <gianna.jordan@sagebase.org>"
+LABEL version=$TAG
```
Comment (linglp) [inline schematic_api/Dockerfile:27]: what is this line doing?

**Comment by GiaJordan** [comment]
> @GiaJordan here it looks like that there are a lot of changes. What are the actual changes?

Enable the "hide whitespace" setting to hide the line ending changes

image

File: schematic_api/Dockerfile (hunk 1 above)
Comment (GiaJordan) [reply schematic_api/Dockerfile:27]: It's taking the value of whatever was passed in as the `TAG` argument and using it as the value for the `version` label
@linglp

**Comment by linglp** [comment]
@GiaJordan Oh I see. That makes sense. I tried validating by building a docker image using your branch, and I could see maintainer info be added: 
Screen Shot 2024-06-11 at 1 28 58 PM

which is different from the current one: 
Screen Shot 2024-06-11 at 1 30 19 PM

**Comment by GiaJordan** [comment]
@linglp yes, the `version` tag is new, and that is expected

**Review by linglp — APPROVED** [review]
approve now. Thanks @GiaJordan

**Pull request merged**